deepchem/feat/roberta_tokenizer.py

```diff
@@ -0,0 +1,63 @@
+from deepchem.feat import Featurizer
+from typing import Dict, List
+try:
+  from transformers import RobertaTokenizerFast
+except ModuleNotFoundError:
+  pass
```

Comment:
You can just raise an exception here. It'll be caught by the try/catch in deepchem/feat/__init__.py

Reply:
If we don't raise the exception we'll get confusing errors on the definition below, so we should raise the exception and also raise an exception below on the test